keps/sig-scheduling/4133-priority-class-value-mutate/README.md

```diff
@@ -0,0 +1,900 @@
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+We'd like priority classes value to be mutable going from release 1.29 because of we're technically achieving the
+same effect with re-creation. Reasons for making priority classes immutable are mentioned in this
+[proposal](https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/pod-priority-api.md#drawbacks-of-changing-priority-classes).
+But we noticed that people are anyways deleting/creating priority classes despite the reasons mentioned above.
+While the deletion and re-creation seems like lesser of both the evils, our experience says that the deletion
+and re-creation of priority classes is more dangerous and error prone, considering people might delete another
+priority class, instead of doing just an update where an update can be rejected if there is a name mismatch or
+value update.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+- Allow mutating priorityClass value
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+- Allow mutating priorityClass other immutable filed like `PreemptionPolicy`
+- Change the priority of running or pending pods when changing the priority value of a priority class
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+
+As a cluster administrator, I wish I could directly adjust the priority class value instead of re-creation.
```

Comment:
I'm not quite convinced by this use case because I hope we can have **a real case** here rather than just emphasizing that I want it. You can explain in which case you want to *adjust the priority class directly* and what we can benefit from it. 

More stories is also welcomed.